src/renderer/before-frame-starts/runnables/setup-root-mac-class.injectable.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import isMacInjectable from "../../../common/vars/is-mac.injectable";
+import { beforeFrameStartsInjectionToken } from "../tokens";
+
+const setupRootMacClassnameInjectable = getInjectable({
+  id: "setup-root-mac-classname",
+  instantiate: (di) => {
+    const isMac = di.inject(isMacInjectable);
+
+    return {
+      id: "setup-root-mac-classname",
+      run: () => {
+        const rootElem = document.getElementById("app");
```

Comment:
Side-effect.

Reply:
By that definition, so it `render`. So I think your point is moot.